static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -130,7 +130,7 @@ function ActionSet({
         openConfirmModal({
           bypass: !onShouldConfirm(ConfirmAction.MERGE),
           onConfirm: onMerge,
-          message: confirm(ConfirmAction.MERGE, false),
+          message: confirm({action: ConfirmAction.MERGE, canBeUndone: false}),
```

Comment:
Changed `confirm` to accept an object instead of positional arguments since I added `statusDetails` and it was already a bit unruly